lib/components/molecules/Table/Table.js

```diff
@@ -0,0 +1,59 @@
+import React, { cloneElement, Children } from 'react';
+import styled from 'styled-components';
+
+import styles from './Table.style';
+// eslint-disable-next-line flowtype/no-types-missing-file-annotation
+import type { Props, ColumnProps } from './types';
+
+// eslint-disable-next-line flowtype/no-types-missing-file-annotation
+const Table = ({ children, data, className, ...props }: Props) => {
+  return (
+    <table className={className} {...props} role="table">
+      <thead>
+        <tr role="row">
```

Comment:
Are you assuming all tables to be Horizontal with header at the top? I had asked this question on earlier PR too.